opm/simulators/wells/StandardWellEval.cpp

```diff
@@ -90,19 +90,19 @@ relaxationFactorFractionsProducer(const std::vector<double>& primary_variables,
     double relaxation_factor = 1.0;
 
     if (FluidSystem::numActivePhases() > 1) {
-        if (FluidSystem::phaseIsActive(FluidSystem::waterPhaseIdx)) {
+        if (has_wfrac_variable) { 
```

Comment:
please use if constexpr for purely compile time conditionals. micro-optim which allows for code elimination.

Reply:
changed `if (...) {` to ` if constexpr (...) { `